client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -130,17 +128,7 @@ class PaymentMethodItem extends Component {
 				/>
 			);
 		}
-		if ( method.id === 'stripe' ) {
-			return (
-				<PaymentMethodStripe
-					method={ currentlyEditingMethod }
-					onCancel={ this.onCancel }
-					onEditField={ this.onEditField }
-					onDone={ this.onDoneAndEnable }
```

Comment:
Can you please also remove the `onDoneAndEnable` method?

Reply:
Sure thing! Done in https://github.com/Automattic/woocommerce-services/pull/2248/commits/ed7b1b779b6bed719596364d3ff8b45f3172768d